lib/Webhooks.js

```diff
@@ -104,6 +104,13 @@ const signature = {
       });
     }
 
+    if (secret.substring(0, 6) != 'whsec_') {
+      throw new StripeSignatureVerificationError({
+        message:
+          'Webhook secret key passed in does not start with whsec_. Webhook secrets can be found here: https://dashboard.stripe.com/webhooks.',
```

Comment:
Suggest rewording this for clarity:

`Wrong signing secret provided. Please ensure you are passing in a valid webhook signing secret that starts with whsec_. Webhook secrets can be found at: https://dashboard.stripe.com/webhooks.`